runtime/astar/src/lib.rs

```diff
@@ -1045,6 +1045,8 @@ pub enum ProxyType {
     StakerRewardClaim,
     /// All governance related calls allowed for proxy account
     Governance,
+    /// All session related calls allowed for proxy account
+    Session,
```

Comment:
Please update the existing `NonTransfer` type instead. Setting or purging keys does not move any funds, so it's fine.

Reply:
@Dinonard I have updated the PR, please review again